idea/idea-gradle-native/src/org/jetbrains/kotlin/ide/konan/gradle/KotlinGradleNativeMultiplatformModuleBuilder.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2010-2018 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.ide.konan.gradle
+
+import org.jetbrains.kotlin.idea.configuration.KotlinGradleAbstractMultiplatformModuleBuilder
+
+class KotlinGradleNativeMultiplatformModuleBuilder : KotlinGradleAbstractMultiplatformModuleBuilder() {
+
+    private var nativeTargetName: String = "ios"
+
+    override fun getBuilderId() = "kotlin.gradle.multiplatform.native"
+
+    override fun getPresentableName() = "Kotlin/Native"
+
+    override fun getDescription() = "Kotlin module for iOS target"
+
+    override val notImportedCommonSourceSets = true
```

Comment:
I'm not sure how valid is the hack related to this property. Hope @asedunov will tell something about it.